dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsAPIImpl.java

```diff
@@ -886,14 +889,19 @@ public void endFinalizedExperiments(final User user) throws DotDataException {
     public void startScheduledToStartExperiments(final User user) throws DotDataException {
         final List<Experiment> scheduledToStartExperiments = list(ExperimentFilter.builder()
                 .statuses(CollectionsUtils.set(SCHEDULED)).build(), user).stream()
-                .filter((experiment -> experiment.scheduling().isPresent() && experiment.scheduling().get().startDate().orElseThrow()
-                        .isAfter(Instant.now()))).collect(Collectors.toList());
+                .filter((experiment -> isTimeReach(experiment.scheduling().get().startDate().orElseThrow())))
+                .collect(Collectors.toList());
 
         scheduledToStartExperiments.forEach((experiment ->
                 Try.of(()->startScheduled(experiment.id().orElseThrow(), user)).getOrElseThrow((e)->
                         new DotStateException("Unable to start Experiment. Cause:" + e))));
     }
-    
+
+    private static boolean isTimeReach(final Instant time) {
```

Comment:
this would be good to have in a DateUtils

Reply:
done https://github.com/dotCMS/core/pull/24218/commits/70723807f1d3d56af3a18143bc79b19245834bf4#diff-f764585bdb3c9ae9a5a0e9932d9e72d46d3996569418db318671348ef2f69143R758